### PR TITLE
[1LP][RFR] Fix test service/ssui tests for cloud provider

### PR DESCRIPTION
**Pull request by niyazRedhat**
PRT Run: 
{{ pytest: cfme/tests/ssui/test_ssui_service_catalogs.py -k test_service_catalog_crud_ssui -vvvv --long-running --use-provider azure --use-provider ec2west --use-provider gce --use-provider openstack }}

All services/ssui cloud test cases were failing while creating the catalog item in 5.9.4
eg: 
http://infra-trackerbot2.cfme2.lab.eng.rdu2.redhat.com/ostriz/dini/?stream_report=downstream-59z&source=jenkins#cfme/tests/ssui/test_ssui_myservice.py/test_myservice_crud[ec2-ViaSSUI]

### CODE REVIEW

File: cfme/fixtures/service_fixtures.py [hunk 1]
```diff
@@ -58,27 +60,49 @@ def create_catalog_item(appliance, provider, provisioning, dialog, catalog,
...
+                    'cloud_network': provisioning['cloud_network']},
+            })
+        # EC2 specific
+        if provider.one_of(EC2Provider):
+            recursive_update(provisioning_data, {
+                'environment': {
+                    'availability_zone': provisioning['availability_zone'],
+                    'cloud_network': provisioning['cloud_network'],
+                    'cloud_subnet': provisioning['cloud_subnet'],
+                    'security_groups': [provisioning['security_group']],
```
Comment (sshveta) [inline cfme/fixtures/service_fixtures.py]: Required : 'security_groups': provisioning['security_group'],

Remove extra bracket

File: cfme/fixtures/service_fixtures.py [hunk 2]
```diff
@@ -58,27 +60,49 @@ def create_catalog_item(appliance, provider, provisioning, dialog, catalog,
...
+                },
+            })
+            # OpenStack specific
+        if provider.one_of(OpenStackProvider):
+            recursive_update(provisioning_data, {
+                'environment': {
+                    'availability_zone': provisioning['availability_zone'],
+                    'cloud_network': provisioning['cloud_network'],
+                    'cloud_tenant': provisioning['cloud_tenant'],
+                    'security_groups': [provisioning['security_group']],
```
Comment (sshveta) [inline cfme/fixtures/service_fixtures.py]: Required : 'security_groups': provisioning['security_group'],

Remove extra bracket

File: cfme/fixtures/service_fixtures.py [hunk 3]
```diff
@@ -58,27 +60,49 @@ def create_catalog_item(appliance, provider, provisioning, dialog, catalog,
...
         if provider.one_of(AzureProvider):
-            recursive_update(provisioning_data, {'customize': {
-                'admin_username': provisioning['customize_username'],
-                'root_password': provisioning['customize_password']},
+            recursive_update(provisioning_data, {
+                'customize': {
+                    'admin_username': provisioning['customize_username'],
+                    'root_password': provisioning['customize_password']},
+                'environment': {
+                    'security_groups': provisioning['security_group'],
```
[the remark this answers is not in the document]
Comment (niyazRedhat) [reply cfme/fixtures/service_fixtures.py]: I have added mandatory fields in `provisioning_data`. I tried with .get() first but I did not got success, then I tried with separate dict and it is working now with the all cloud providers. I checked by running other tests with GCE, EC2, Azure and Openstack provider and all other tests don't have any effect of this.

About failures, looks like it is selenium issue. I have data in dict, and selenium don't wait for tab get loaded and it fails with NoSuchElementException. Thing is, we can not add wait_for in dict value, so sometimes when tab get loaded it passes but when it doesn't test fails.

File: cfme/fixtures/service_fixtures.py [hunk 4]
```diff
@@ -58,27 +60,49 @@ def create_catalog_item(appliance, provider, provisioning, dialog, catalog,
...
+                    'availability_zone': provisioning['availability_zone'],
+                    'cloud_network': provisioning['cloud_network'],
+                    'cloud_subnet': provisioning['cloud_subnet'],
+                    'security_groups': provisioning['security_group'],
+                },
+            })
+            # OpenStack specific
+        if provider.one_of(OpenStackProvider):
+            recursive_update(provisioning_data, {
+                'environment': {
```
[the remark this answers is not in the document]
Comment (niyazRedhat) [reply cfme/fixtures/service_fixtures.py]: please read above comments.

**Pull request merged**